operator/webserver.py

```diff
@@ -41,33 +43,62 @@ def launch_simulator(self):
         if self.simulation_process is not None:
             abort(400, description='Simulation already running.')
 
+        print("Creating logfile for run")
+
         # Get team config
         teamId = request.json['id']
         self.mission = request.json['mission']
+        track = request.json['track']
         for obj in self.team_config['teams']: 
             if obj['id'] == teamId: self.team = obj  
 
+        # Create log file. Create logs directory if it does not exist
+        filename = 'logs/{}_{}_{}.txt'.format(self.team['name'].lower().replace(' ', '-'), self.mission, str(datetime.now().strftime("%d-%m-%Y_%Hh%Mm%Ss")))
```

Comment:
create logfile during simulator launch instead of mission launch so that mission launch is less prone to error.